fractal_server/app/routes/api/v2/task.py

```diff
@@ -66,6 +75,25 @@ async def get_list_task(
     if args_schema_non_parallel is False:
         for task in task_list:
             setattr(task, "args_schema_non_parallel", None)
+    if authors is not None:
+        task_list = [
+            task
+            for task in task_list
+            if all(
+                author.lower()
+                in (t_author.lower() for t_author in task.authors)
+                for author in authors
+            )
+        ]
+    if tags is not None:
```

Comment:
Let's postpone implementing this query, so that we can take our time to learn aboute sqlalchemy/sqlite/postgres support for JSON queries.
(please open a new issue)

Reply:
https://github.com/fractal-analytics-platform/fractal-server/issues/1863